src/composer.json

```diff
@@ -32,5 +35,11 @@
             "Fossology\\Decider\\UI\\": "decider/ui",
             "Fossology\\UI\\Page\\": "www/ui/page"
         }
+    },
+    
+    "config": {
+        "platform": {
+            "php": "5.3.3"
```

Comment:
I agree with you and I would suggest to drop 5.3 now. We also have released 3.0.0 for 5.3 as suggested in the issue #585 and the next one will be with new features.

@mcjaeger: what is your opinion?